doc/ClangFormat.md

```diff
@@ -80,6 +119,8 @@ TODO
 
 ## TODOs
 
+- [X] Add travis test file
+- [X] Enable to lint one's work offlien - prior to PR/commit
 - [ ] TODO - update the links
 - [ ] Talk about the reformat_branch
 - [ ] Solve all TODOs (how meta! :-) )
```

Comment:
>What links do you refer here to?

Links of this sorts:

```sh
[check_style.sh script](TODO - Add link)
```

But its easier to update them after merging this because then I'll know either the exact URL for the file or the commit ID in similar cases...